metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughAPITest.java

```diff
@@ -114,9 +115,17 @@ public void importReferenceSequenceObject() throws Exception {
         assertEquals("EquCab2.0", referenceSequence.getName());
         assertEquals(ReferenceSequence.Type.ASSEMBLY, referenceSequence.getType());
         Taxonomy taxonomy = referenceSequence.getTaxonomy();
-        assertEquals(9796, taxonomy.getTaxonomyId().longValue());
+        assertEquals(9796, taxonomy.getTaxonomyId());
         assertEquals("Equus caballus", taxonomy.getName());
         assertEquals(1, referenceSequenceRepository.count());
     }
 
+    @Test
+    public void importSampleObject() throws Exception {
+        Sample sample = sraObjectImporter.importSample("ERS000156");
```

Comment:
Can you assert the sample count in import Study as well?

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/55/commits/43933133c552f45457607c7997255db1a4c4755b